tests/baselines/reference/config/showConfig/Shows tsconfig for single option/rewriteRelativeImportExtensions/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+    "compilerOptions": {
+        "rewriteRelativeImportExtensions": true,
+        "allowImportingTsExtensions": true
```

Comment:
I think something's wrong for `showConfig` - it's supposed to be smart enough to know `allowImportingTsExtensions` is derived from `rewriteRelativeImportExtensions` and elide it, right?

Reply:
No, since a couple releases ago, `--showConfig` shows all implied options (if they evaluate to something other than their default value)